lib/elixir_conf_africa/events.ex

```diff
@@ -21,6 +22,26 @@ defmodule ElixirConfAfrica.Events do
     Repo.all(from e in Event, order_by: [desc: e.id])
   end
 
+  def get_elixir_conf_event_and_ticket_types do
+    get_elixir_conf_event()
+    |> Repo.preload(:ticket_types)
+  end
+
+  defp get_elixir_conf_event do
+    Repo.get_by(Event, name: "ElixirConf Africa 2024")
+  end
+
+  def get_all_available_tickets do
+    get_elixir_conf_event = get_elixir_conf_event()
```

Comment:
you can also get the event in the query below using a join. This way you only hit the db once

Reply:
I have done this